src/main/java/tasks/Todo.java

```diff
@@ -0,0 +1,43 @@
+package main.java.tasks;
+import main.java.exceptions.InvalidDescriptionException;
```

Comment:
Shall we add a blank line between the package statement and the import statement here?